code/src/components/StartGame.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { startNewGame } from 'reducers/game';
+import swal from "sweetalert";
+import styled from 'styled-components'
```

Comment:
Should be written like this:

import React from 'react'
import { useDispatch, useSelector } from 'react-redux'
import swal from "sweetalert";
import styled from 'styled-components'

import { startNewGame } from 'reducers/game';